schema/sql/settings.sql

```diff
@@ -0,0 +1,10 @@
+SET statement_timeout = 0;
```

Comment:
We don't want this file

Reply:
This is here because this is exactly how production is setup right now, and we were not auditing this otherwise. Be explicit.